app/routes/reductions.js

```diff
@@ -0,0 +1,76 @@
+const organisationUnitConstants = require('../constants/organisation-unit')
+const reductionsService = require('../services/reductions-service')
+const Reduction = require('../services/domain/new-reduction')
+
+module.exports = function (router) {
+  router.get('/:organisationLevel/:id/reductions', function (req, res, next) {
+    var organisationLevel = req.params.organisationLevel
+    var id = req.params.id
+    var successText = req.body.successText ? req.params.successText : null
+
+    if (organisationLevel !== organisationUnitConstants.OFFENDER_MANAGER.name) {
+      throw new Error('Only available for offender manager')
+    }
+
+    var reductionsPromise = reductionsService.getReductions(id)
```

Comment:
Need to pass `organisationalLevel` here also?

Reply:
Yea, good spot. SHould be in, in the latest push.